src/nmodl/main.cpp

```diff
@@ -307,7 +314,8 @@ int main(int argc, const char* argv[]) {
             // run perfvisitor to update read/wrie counts
             PerfVisitor().visit_program(*ast);
             // If there is an incompatible construct and code generation is not forced exit NMODL
-            if (CodegenCompatibilityVisitor().find_unhandled_ast_nodes(*ast) && !force_codegen) {
+            if (CodegenCompatibilityVisitor(nmodl_global_to_range).find_unhandled_ast_nodes(*ast) &&
+                !force_codegen) {
```

Comment:
You're right 👍  It would be even better probably if we first run the `GlobalToRange` visitor and then pass a flag to enable the error, given the logic I described in above comment